mambaSharedFramework/HLS Models/VariantPlaylist.swift

```diff
@@ -201,3 +177,10 @@ extension HLSPlaylistInterface {
         return locationTag.tagData.stringValue()
     }
 }
+
+/**
+ This is a protocol that defines the standard `VariantPlaylist` interface.
+ 
+ This protocol should be used instead of the actual `VariantPlaylist` type when possible.
```

Comment:
same question as for master.

Reply:
Same answer. Nothing special, just good design.